### PR TITLE
feat(formatter): liquidsoap-prettier

### PR DESCRIPTION
**Pull request by elythh**
A formatter for https://www.liquidsoap.info/ 😄

### CODE REVIEW

**Comment by elythh** [comment]
I think I should rather configure prettier to use that plugin.

File: lua/conform/formatters/liquidsoap-prettier.lua [hunk 1]
```diff
@@ -0,0 +1,10 @@
+local util = require("conform.util")
+---@type conform.FileFormatterConfig
+return {
+  meta = {
+    url = "https://github.com/savonet/liquidsoap-prettier",
+    description = "A binary to format Liquidsoap scripts",
+  },
+  command = util.from_node_modules("liquidsoap-prettier"),
+  args = { "$FILENAME" },
```
Comment (stevearc) [inline lua/conform/formatters/liquidsoap-prettier.lua:9]: This command is writing the formatting to stdout, but it's reading it from the file. I think you need to set
```lua
args = { "-w", "$FILENAME" },
stdin = false,
```

**Review by stevearc — CHANGES_REQUESTED** [review]

File: lua/conform/formatters/liquidsoap-prettier.lua (hunk 1 above)
Comment (elythh) [reply lua/conform/formatters/liquidsoap-prettier.lua]: Both work but that way I cleaner ye

**Comment by stevearc** [comment]
Thanks for the PR!

**Pull request merged**